packages/scripts/src/addOrbitChain/tests/transforms.test.ts

```diff
@@ -193,4 +195,103 @@ describe("Transforms", () => {
       expect(resizedAspectRatio).toBeCloseTo(originalAspectRatio, 2);
     });
   });
+  describe("Image Download and Processing", () => {
+    const downloadedImagePath = path.join(
+      process.cwd(),
+      "..",
+      "..",
+      "arb-token-bridge-ui",
+      "public",
+      "images",
+      "downloaded_chain_logo.png"
+    );
+
+    // Clean up downloaded image after tests
+    // Comment out the following 'after' block if you want to inspect the downloaded image
+    afterAll(() => {
+      if (fs.existsSync(downloadedImagePath)) {
+        fs.unlinkSync(downloadedImagePath);
+        console.log("Cleaned up downloaded image");
+      }
+    });
+
+    it("should download, process, and save the chain logo image from fullMockIssue", async () => {
+      const rawChainData = extractRawChainData(fullMockIssue);
+      const imageUrl = rawChainData.chainLogo as string;
+
+      expect(imageUrl).toBeTruthy();
+      expect(imageUrl.startsWith("https://")).toBe(true);
+
+      const { buffer, fileExtension } = await fetchAndProcessImage(imageUrl);
+
+      expect(buffer).toBeTruthy();
+      expect(buffer.length).toBeGreaterThan(0);
+      expect(fileExtension).toBeTruthy();
+
+      const fileName = "downloaded_chain_logo";
+      const savedImagePath = saveImageLocally(buffer, fileName, fileExtension);
+
+      expect(savedImagePath).toBeTruthy();
+      console.log(`Image downloaded and saved to: ${savedImagePath}`);
+
+      const fullSavePath = path.join(
+        process.cwd(),
+        "..",
+        "..",
+        "arb-token-bridge-ui",
+        "public",
+        savedImagePath
+      );
+      expect(fs.existsSync(fullSavePath)).toBe(true);
+
+      const stats = fs.statSync(fullSavePath);
+      expect(stats.size).toBeGreaterThan(0);
+    });
+
+    it("should download, process, and save the chain logo image from IPFS URL", async () => {
+      const ipfsUrl = "ipfs://QmYAX3R4LhoFenKsMEq6nPBZzmNx9mNkQW1PUwqYfxK3Ym";
```

Comment:
can you also add a test for `https://ipfs.io/ipfs/QmYAX3R4LhoFenKsMEq6nPBZzmNx9mNkQW1PUwqYfxK3Ym` type of url?